Gopkg.toml

```diff
@@ -30,11 +30,11 @@
   name = "github.com/cenkalti/backoff"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```

Comment:
This is to be reverted once https://github.com/giantswarm/e2e-harness/pull/209 gets merged.